rdrf/rdrf/admin.py

```diff
@@ -556,12 +557,15 @@ class RegistryYamlAdmin(admin.ModelAdmin):
 ]
 
 PROMS_ADMIN_COMPONENTS = [
-    (RegistryYaml, RegistryYamlAdmin),
```

Comment:
put back 

PROMS_ADMIN_COMPONENTS = [
    (RegistryYaml, RegistryYamlAdmin),


seems worthwhile to keep after all